ApiSurface/ApiSurface.fs

```diff
@@ -239,12 +243,14 @@ module ApiSurface =
         else
             printfn "Updated version to %d.%d" updatedVersion.Major updatedVersion.Minor
 
-        writeUpdatedVersion updatedVersion versionFile
+        do
+            use versionFile = openVersionFile versionFile (FileMode.Truncate, FileAccess.Write)
+            writeUpdatedVersion oldVersionFile updatedVersion versionFile
 
     let writeAssembly
         (baseline : ApiSurface)
         (possibleBaselineResources : string list)
-        (versionFiles : IFileInfo list)
+        (versionFiles : FileInfo list)
```

Comment:
This isn't on the API surface and the IFileSystem was never instantiated with anything except a real file system.